augly/tests/video_tests/transforms/composite_test.py

```diff
@@ -74,11 +87,10 @@ def test_RandomPixelization(self):
     def test_ReplaceWithBackground(self):
         self.evaluate_class(
             vidaugs.ReplaceWithBackground(
-                source_offset=0.1,
-                background_offset=0,
-                source_percentage=0.7,
+                source_offset=0.1, background_offset=0, source_percentage=0.7,
             ),
             fname="replace_with_background",
+            metadata_exclude_keys=["dst_duration", "dst_fps"],
```

Comment:
This test was failing for both him & me on our local envs, which is just another symptom of our tests being brittle to slight changes in ffmpeg version etc so I thought it might be better to just ignore the fields which are different between the envs. Lmk if you have any other thoughts!

Reply:
sgtm!